pages/datasets/_datasetId.vue

```diff
@@ -860,22 +860,56 @@ export default {
   },
   methods: {
     /**
-     * Returns protocol records in a dataset's model if they exist
+     * Sets active tab
+     * @param {String} activeLabel
+     */
+    setActiveTab: function(activeLabel) {
```

Comment:
Is this function used anywhere?

Reply:
yes, it is used as a call-back on line 162 I think.